x-pack/plugins/lens/public/embeddable/embeddable.tsx

```diff
@@ -582,6 +606,13 @@ export class Embeddable
     // the container to pick them up and use them to configure filter bar and
     // config dropdown correctly.
     const input = this.getInput();
+
+    this.errors = this.maybeAddTimeRangeError(this.errors, input, indexPatterns);
+
+    if (this.errors) {
+      this.logError('validation');
```

Comment:
I know you just moved it from different place, but if this is logged to the user, maybe we could log a nicer message? 🤔

Reply:
I think this just indicates the type of error, not the actual message. 
IKf you think this error message can be improved I agree 😅 : https://github.com/elastic/kibana/pull/123587/files#diff-06b33562b1971e1caf72a3a592b2ecd93a4b7199fe9d1bf66f36bc9d906325c1R326